test/integration/create_user_test.rb

```diff
@@ -7,12 +7,14 @@ class CreateUserTest < ActionDispatch::IntegrationTest
     assert_response :success
     assert_difference 'User.count', 1 do
       post users_path, params: {user: { username: "Victoria", email: "victoria@email.com", 
-                                        password: "password", password_confirmation: "password" } }
+                                        password: "password", password_confirmation: "password", 
+                                        email_confirmed: true} }
     end
     assert_response :redirect
-    assert_redirected_to articles_url
+    assert_redirected_to root_path
     follow_redirect!
-    assert_match "Victoria", response.body
+    assert_select "div.alert"
+    assert_match "Please check your emails to confirm your sign up", response.body
```

Comment:
Also *wenn* Du Lust hast: Hier könnten wir den Test noch weiterführen, so dass wir in die Email reinschauen, dort den Link raussuchen und klicken und dann gucken, ob wir eingeloggt sind...

:-)